### PR TITLE
init: Ask to login if not already

### PR DESCRIPTION
**Pull request by cheungpat**
(merge #23 first then cherry-pick this)

### CODE REVIEW

File: src/commands/init.js [hunk 1]
```diff
@@ -15,17 +15,20 @@
...
 import { controller } from '../api';
 import * as config from '../config';
+import { createCommand } from '../util';
 import * as template from '../template';
+import LoginCommand from './login';
 
 function ensureLoggedIn(argv) {
   if (!argv.currentAccount) {
-    return Promise.reject('Requires authentication, please run skycli login.');
+    console.log(chalk.yellow('Requires authentication, please run login.'));
```
Comment (rickmak) [inline src/commands/init.js]: I think `Requires authentication, please login.` is better, since we prompt for email immediately.

**Review by rickmak — CHANGES_REQUESTED** [review]
`ensureLoggedIn` should be at utils and `deploy` command also need the check.

File: src/commands/init.js (hunk 1 above)
Comment (cheungpat) [reply src/commands/init.js]: oh... forgot to delete that word

**Comment by cheungpat** [comment]
@rickmak thanks! updated

**Comment by rickmak** [comment]
Merge via 9edc52a

**Pull request closed without merging**